index.html

```diff
@@ -24,202 +22,221 @@
-            class="separator"
-        /></span>
-        <a
-          href="https://www.facebook.com/Real-Dev-Squad-108713777585062/"
-          target="_blank"
-          class="social-handles"
-          id="facebook"
-          >Facebook
-        </a>
-        <a
-          href="https://www.facebook.com/Real-Dev-Squad-108713777585062/"
-          target="_blank"
-        >
-          <img
-            src="../img/re-designed-images/facebook-logo.png"
-            alt="Facebook logo"
-            class="handles-logos"
-          />
-        </a>
+            <a
+              href="https://twitter.com/realdevsquad?lang=en"
+              target="_blank"
+              class="social-handles"
+              id="twitter"
+            >
+              Twitter
+              <img
+                src="../img/re-designed-images/twitter-logo.png"
+                alt="Twitter_Logo"
+              />
+            </a>
 
-        <span
-          ><img
-            src="../img/re-designed-images/separator-icon.png"
-            alt="separator icon"
-            class="separator"
-        /></span>
-        <a
-          href="https://www.instagram.com/realdevsquad/?hl=en"
-          target="_blank"
-          class="social-handles"
-          id="instagram"
-          >Instagram
-        </a>
-        <a href="https://www.instagram.com/realdevsquad/?hl=en" target="_blank">
-          <img
-            src="../img/re-designed-images/instagram-logo.png"
-            alt="Instagram logo"
-            class="handles-logos"
-          />
-        </a>
+            <span>
+              <img
+                src="../img/re-designed-images/separator-icon.png"
+                alt="Separator_Icon"
+                class="separator"
+              />
+            </span>
+
+            <a
+              href="https://www.facebook.com/Real-Dev-Squad-108713777585062/"
+              target="_blank"
+              class="social-handles"
+              id="facebook"
+            >
+              Facebook
+              <img
+                src="../img/re-designed-images/facebook-logo.png"
+                alt="Facebook_Logo"
+              />
+            </a>
+
+            <span>
+              <img
+                src="../img/re-designed-images/separator-icon.png"
+                alt="Separator_Icon"
+                class="separator"
+              />
+            </span>
+
+            <a
+              href="https://www.instagram.com/realdevsquad/?hl=en"
+              target="_blank"
+              class="social-handles"
+              id="instagram"
+            >
+              Instagram
+              <img
+                src="../img/re-designed-images/instagram-logo.png"
+                alt="Instagram_Logo"
+              />
+            </a>
+          </div>
+        </span>
+        <img
+          class="welcome-image"
+          src="../img/re-designed-images/welcome-image.png"
+          alt="Welcome_Image"
+        />
       </div>
 
```

Comment:
![image](https://user-images.githubusercontent.com/66248039/147713592-a36e427e-76a6-40a7-9c33-b2491c8f826b.png)
Can we increase the space between them for mobile screens? Its looking a little crowded.

Reply:
Yes sure